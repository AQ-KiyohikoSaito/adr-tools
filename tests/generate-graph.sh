adr init
adr new An idea that seems good at the time
adr new -s 2 A better idea
adr new This will work
adr new -s 3 The end
adr generate graph
