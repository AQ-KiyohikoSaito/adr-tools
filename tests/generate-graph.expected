adr init
doc/adr/0001-record-architecture-decisions.md
adr new An idea that seems good at the time
doc/adr/0002-an-idea-that-seems-good-at-the-time.md
adr new -s 2 A better idea
doc/adr/0003-a-better-idea.md
adr new This will work
doc/adr/0004-this-will-work.md
adr new -s 3 The end
doc/adr/0005-the-end.md
adr generate graph
digraph {
  node [shape=plaintext];
  _1 [label="1. Record architecture decisions"; URL="0001-record-architecture-decisions.html"]
  _2 [label="2. An idea that seems good at the time"; URL="0002-an-idea-that-seems-good-at-the-time.html"]
  _1 -> _2 [style="dotted"];
  _2 -> _3 [label="Superceded by"]
  _3 [label="3. A better idea"; URL="0003-a-better-idea.html"]
  _2 -> _3 [style="dotted"];
  _3 -> _2 [label="Supercedes"]
  _3 -> _5 [label="Superceded by"]
  _4 [label="4. This will work"; URL="0004-this-will-work.html"]
  _3 -> _4 [style="dotted"];
  _5 [label="5. The end"; URL="0005-the-end.html"]
  _4 -> _5 [style="dotted"];
  _5 -> _3 [label="Supercedes"]
}
